netty/src/main/java/io/grpc/netty/Utils.java

```diff
@@ -63,6 +64,7 @@
 /**
  * Common utility methods.
  */
+@VisibleForTesting
```

Comment:
It's package-private, so no need for `@Internal`. The Netty package is exposed to our users (and they can use it directly); it is entirely `@ExperimentalApi` though.